src/Components/Archived-card/ArchivedCard.jsx

```diff
@@ -0,0 +1,26 @@
+import "../../css/main.css";
+import "../Display-card/displaycard.css";
+import { useArchive } from "../../Context";
+
+export const ArchivedCard = ({item}) => {
+    const { _id, title, description, tag, priority, selectedBackgroundColor } = item;
+    const { deleteFromArchive, restoreFromArchive } = useArchive();
+    return(
+        <>
+        <div className="display-card flex flex-column pd-1" style={{backgroundColor: selectedBackgroundColor}} key={_id}>
+            <div className="container-input-text pdb-1">
+                <h5 className="pdb-1">{title}</h5>
+                <p className="pdb-1 text-display-card text-base">{description}</p>
+            </div>
+            <div className="edit-section flex flex-justify-space-between">
+                <div className="tag">{tag}</div>
+                <div className="priority">{priority}</div>
+                <button className="btn-transparent" onClick={() => {restoreFromArchive(_id)}}><span className="material-icons-outlined btn-archive">unarchive</span></button>
```

Comment:
suggestion: use code formatting tools and editor config file, to avoid lengthy lines and format

Reply:
Done.